troubleshooting.md

```diff
@@ -0,0 +1,550 @@
+# Troubleshooting Guide for BM IPI Deployments
+
+
+* [Troubleshooting Overview](#troubleshooting-overview)
+
+* [Bootstrap VM issues](#bootstrap-vm-issues)
+
+* [Inspecting Logs](#inspecting-logs)
+
+* [Following the Installation](#following-the-installation) 
+
+* [Post install pod errors](#post-install-pod-errors)
+
+* [NTP out of sync](#ntp-out-of-sync)
+
+* [Registry Issues](#registry-issues) 
+
+* [Cleaning up previous installations](cleaning-up-previous-installations)
+
+* [Misc Issues](#misc-issues) 
+
+
+# Troubleshooting Overview
+
+When attempting to deploy Installer Provisioned Infrastructure (IPI) of OpenShift on Bare Metal (BM) you may run into a situation where you need to troubleshoot your environment. This document provides troubleshooting guidance and tips in solving common issues that may arise. 
```

Comment:
When -> While ?

Reply:
Added "While", instead of "When" as suggested.